README.md

```diff
@@ -148,6 +142,70 @@ Use Qt Creator as before
 </pre>
 
 
+Usage
+=====
+
+First use - optimise
+--------------------
+
+It is strongly recommended to run the `volk_profile` utility before
+running gqrx. This will detect and enable processor-specific optimisations and
+will in many cases give a significant performance boost.
+
+The first time you start gqrx it will open a device configuration dialog.
+Supported devices that are connected to the computer are discovered
+automatically and you can select any of them in the drop-down list.
+
+If you don't see your device listed in the drop-down list it could be because:
+- The driver has not been included in a binary distribution
+- The udev rule has not been properly configured
+- Linux kernel driver is blocking access to the device
+
+You can test your device using device specific tools, such as rtl_test,
+airspy_rx, hackrf_transfer, qthid, etc.
+
+Gqrx supports multiple configurations and sessions if you have several devices
+or if you want to use the same device under different configurations. You can
+load a configuration from the GUI or using the `-c` command line argument. See
+`gqrx --help` for a complete list of command line arguments.
+
+Tutorials and howtos
+---------------------
+
+Tutorials and howtos are being written and published on the website
+https://gqrx.dk/
+
+
+Known problems
+--------------
+
+See the bug tracker on Github: https://github.com/gqrx-sdr/gqrx/issues
+
+
```

Comment:
It might be worth to give some details on which data should be provided for bug-reports

Reply:
Yup. I think best approach to this would be to create a bug report issue template? Thoughts? Before I go create one?

Reply:
Maybe we could create a small script which would collect most of the required info?
This cript could run uname-a, etc

Reply:
I think, good bug report should look like this:
Title: `BUG: Gqrx freezes when hackrf (selected as an input device) is disconnected from USB port.`
Body:
```
GUI freeze when HackRF is selected as an input device and USB connection breaks.

Conditions:
gqrx built from git master
Platform: Linux
Distribution: Ubuntu ...
GNU Radio v3.9...
Qt: 5.9.5...
Libhackrf: ...

Steps to reproduce:
1. Connect HackRF to a USB port
2. Start gqrx
3. Select HackRF as an input device and start the DSP
4. Disconnect HackRF from the USB port

Expected behavior:
Gqrx should stop producing audio samples and should continue to show last FFT result. Gqrx should allow to select another input device using the "Configure IO devices" dialog.

Observed behavior:
Gqrx GUI freezes. No reaction to "Configure IO devices" button.

Screenshots, videos, additional notes, stack traces...
```